marray/_mask_string.py

```diff
@@ -0,0 +1,65 @@
+import re
+
+# Regular expression generated by ChatGPT
```

Comment:
Just to be clear, the regular expressions are the *only* thing written by ChatGPT. I avoid relying on it, but I don't love writing regular expressions. I've learned and forgotten the syntax too many times.